src/dbally/audit/event_tracker.py

```diff
@@ -92,3 +92,15 @@ async def track_event(self, event: Event) -> AsyncIterator[EventSpan]:
             await handler.event_end(
                 span.data, event_context=contexts[handler], request_context=self._request_contexts[handler]
             )
+
+    async def log_message(self, message: str, log_level: LogLevel = LogLevel.INFO) -> None:
```

Comment:
@micpst wdyt?

Reply:
I like the idea of `FallbackEvent`. I have some doubts about passing `EventTracker` to execute the next collection in the sequence. This can cause confusion when two collections have different event handlers, as we'll overwrite the collection's handlers down the chain. Maybe we should think about creating a span here to log events from a given chain, and each subchain would have its own subspan to handle, which will end up with a nice trace. For different event handlers, we'll have just a span for a collection node, with enough information. This is a high-level idea, but I think it could work.